### ISSUE TITLE
If you add another two convolutional layers and a max pulling layer with dropout you can seriously improve the underfitting of the cifar10_cnn model

### ISSUE THREAD

**Issue by AlexGidiotis** [fchollet/keras#4749]
Please make sure that the boxes below are checked before you submit your issue. Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by AlexGidiotis** [comment]
#==================== Build the Model =========================
model = Sequential()

model.add(Convolution2D(32, 3, 3, border_mode='full', input_shape=X_train.shape[1:]))
model.add(Activation('relu'))
model.add(Convolution2D(32, 3, 3))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2,2)))
model.add(Dropout(0.25))

model.add(Convolution2D(64, 3, 3, border_mode='full'))
model.add(Activation('relu'))
model.add(Convolution2D(64, 3, 3))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2,2)))
model.add(Dropout(0.25))

model.add(Convolution2D(128, 3, 3, border_mode='full'))
model.add(Activation('relu'))
model.add(Convolution2D(128, 3, 3))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2,2)))
model.add(Dropout(0.25))

model.add(Flatten())
model.add(Dense(512))
model.add(Activation('relu'))
model.add(Dropout(0.5))
model.add(Dense(NB_CLASSES))
model.add(Activation('softmax'))

#===================== Compile the Model ================================
model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])

**Comment by fchollet** [comment]
Yes, if you add more layers you can get a better model. That's besides the point, which was to have a simple example to demonstrate image classification.
